src/Commands/Workflow/InfoCommand.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Workflow;
+
+use Pantheon\Terminus\Commands\MultiCommand;
+
+class InfoCommand extends MultiCommand
+{
+    /**
+     * Show all details for a workflow including operations.
+     *
+     * @command workflow:info
+     *
+     * @param string $site_id Site name that the workflow is part of.
+     *
+     * @option string $workflow-id ID of the workflow to show.
+     * @option string $latest-with-logs Just show the latest workflow with logs.
+     *
+     * @usage terminus workflow:info <site_name> <wf-id>
+     *   Show the workflow with id <wf-id> found on site <site_name>.
+     */
+    public function info($site_id, $options = ['latest-with-logs' => false, 'workflow-id' => ''])
+    {
+        $this->subCommand('workflow:info:status', $this->getInput(), $this->getOutput());
```

Comment:
`$this->getInput()` and `$this->getOutput()` are available to the subclass, so you wouldn't have to pass these as parameters.  I'm ambivalent here.  It's cleaner to not pass parameters needlessly; however, doing it this way has a couple of advantages:

- Testing the `subCommand` function is a little more straightforward
- The caller of `subCommand` might want to capture output or modify the input object first

The former is only a slight distinction; as for the later, I'm not sure that it's a use case we need to support.

Up to @TeslaDethray; not sure which way I ultimately recommend.  I'd prefer the shorter variant if there's no cause for doing it this way.